task/buildah-remote-oci-ta/0.2/buildah-remote-oci-ta.yaml

```diff
@@ -204,10 +204,11 @@ spec:
     - $(params.BUILD_ARGS[*])
     computeResources:
       limits:
-        memory: 4Gi
+        cpu: "4"
+        memory: 8Gi
       requests:
-        cpu: 250m
-        memory: 512Mi
+        cpu: "1"
+        memory: 2Gi
```

Comment:
I am OK to keep this in the change but I want to mention that for remote build, the pod in ocp does not do much as the build is happening in a remote VM.  We can keep it like that as it will help the scheduler to spread pods on nodes.

Reply:
Oh, right. We should set requests/limits for buildah-remote task in build-definitions so we are not wasting resources.